package.json

```
@@ -15,6 +15,9 @@
     "path"
   ],
   "author": "Per Liedman <per@liedman.net>",
+  "contributers": [
```

Should be:

```suggestion
  "contributors": [
```

...right? (https://docs.npmjs.com/files/package.json#people-fields-author-contributors)